load.php

```diff
@@ -410,7 +423,10 @@ function perflab_maybe_set_object_cache_dropin() {
 		$dropin_path = WP_CONTENT_DIR . '/object-cache.php';
 
 		/**
```

Comment:
While we're here, multi-line inline comments should not use two asterisks

```suggestion
		/*
```

Reply:
Done in https://github.com/WordPress/performance/pull/1047/commits/56ec805a7925c6223394a547fba8c2e28b8b91e7